servers/publikator/lib/translations.json

```diff
@@ -57,6 +57,14 @@
     {
       "key": "api/twitter/morephotos/1",
       "value": "1 weiteres Foto"
+    },
+    {
+      "key": "api/publish/disabled",
+      "value": "Artikel können im Moment nicht veröffentlicht werden."
```

Comment:
just go with `Artikel können im Moment weder veröffentlicht noch zurückgezogen werden. Bitte versuche es in ein paar Minuten nochmals, ansonsten kontaktiere bitte die IT.`